apps/publish/content/common.py

```diff
@@ -538,28 +542,46 @@ def apply_kill_override(self, item, updates):
         except Exception:
             logger.exception('Failed to apply kill header template to item {}.'.format(item))
 
-    def _refresh_associated_items(self, original):
+    def _refresh_associated_items(self, original, updates={}, publish=False):
         """Refresh associated items before publishing
 
         Any further updates made to basic metadata done after item was associated will be carried on and
         used when validating those items.
         """
+        publish_service = None
+        if publish_services.get(self.publish_type):
+            publish_service = get_resource_service(publish_services[self.publish_type])
+
         associations = original.get(ASSOCIATIONS) or {}
-        for _, item in associations.items():
+        for associations_key, item in associations.items():
             if type(item) == dict and item.get(config.ID_FIELD):
                 keys = DEFAULT_SCHEMA.keys()
                 if app.settings.get('COPY_METADATA_FROM_PARENT') and item.get(ITEM_TYPE) in MEDIA_TYPES:
-                    updates = original
+                    original_item = original
                     keys = FIELDS_TO_COPY_FOR_ASSOCIATED_ITEM
                 else:
-                    updates = super().find_one(req=None, _id=item[config.ID_FIELD]) or {}
+                    original_item = super().find_one(req=None, _id=item[config.ID_FIELD]) or {}
 
                 try:
-                    is_db_item_bigger_ver = updates['_current_version'] > original['_current_version']
+                    is_db_item_bigger_ver = original_item['_current_version'] > original['_current_version']
```

Comment:
`original_item['_current_version'] > original['_current_version']` always be `false` as we are comparing the same item. If you have the config setting `COPY_METADATA_FROM_PARENT` and it won't copy/overwrite the `metadata` from the `item` to the `associated media`.

@mugurrus these changes were done as part of SDFID-211.

Reply:
I just renamed the variable, didn't change the logic here.

Reply:
@mugurrus I know that. Hence, I have stated that in my comment above. Would it be possible to change it?

Reply:
@mdhaman How would you like to change it?

Reply:
@mugurrus thanks. Logic is correct now as you have removed `is_db_item_bigger_ver 
 = original_item['_current_version'] > original['_current_version']` check.